src/main/java/com/vkatit/cinema/log/Log.java

```diff
@@ -0,0 +1,69 @@
+package com.vkatit.cinema.log;
```

Comment:
Log has to be configured using only configuration  file. Don't write any code for it.

Reply:
Done